[2024-04-16T09:41:55.369+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-16T09:41:55.395+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spotify_final_dag.hello scheduled__2024-04-16T06:29:00+00:00 [queued]>
[2024-04-16T09:41:55.406+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spotify_final_dag.hello scheduled__2024-04-16T06:29:00+00:00 [queued]>
[2024-04-16T09:41:55.407+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-16T09:41:55.426+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): hello> on 2024-04-16 06:29:00+00:00
[2024-04-16T09:41:55.439+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3483) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-16T09:41:55.441+0000] {standard_task_runner.py:63} INFO - Started process 3514 to run task
[2024-04-16T09:41:55.441+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'spotify_final_dag', 'hello', 'scheduled__2024-04-16T06:29:00+00:00', '--job-id', '880', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpfu3nq16p']
[2024-04-16T09:41:55.443+0000] {standard_task_runner.py:91} INFO - Job 880: Subtask hello
[2024-04-16T09:41:55.498+0000] {task_command.py:426} INFO - Running <TaskInstance: spotify_final_dag.hello scheduled__2024-04-16T06:29:00+00:00 [running]> on host 52651c1f9987
[2024-04-16T09:41:55.611+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spotify_final_dag' AIRFLOW_CTX_TASK_ID='hello' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T06:29:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-16T06:29:00+00:00'
[2024-04-16T09:41:55.622+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-16T09:41:59.196+0000] {logging_mixin.py:188} INFO - There was a problem with the connection. Trial number 1, Status Code: 401
[2024-04-16T09:42:02.467+0000] {logging_mixin.py:188} INFO - There was a problem with the connection. Trial number 2, Status Code: 401
[2024-04-16T09:42:02.469+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-16T09:42:02.470+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-16T09:42:02.477+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=spotify_final_dag, task_id=hello, execution_date=20240416T062900, start_date=20240416T094155, end_date=20240416T094202
[2024-04-16T09:42:02.515+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-16T09:42:02.528+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-16T09:42:02.529+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
